groupsforsections.py

```diff
@@ -255,7 +283,39 @@ def main():
 
         # mapping all the users in a sections to corresponding group
         groups_to_users_dict[group_id] = users
-        # TODO: Adding users to corresponding groups
+
+    failed_groups_to_users_dict={}
+    success_groups_to_users_dict={}
+
+    # adding users to the group
+    for group in groups_to_users_dict:
+        for user in groups_to_users_dict[group]:
```

Comment:
I like the pythonic way of doing this, I changed the `for` loop as suggested as it is more clear and readable than showing it like `groups_to_users_dict[group]`. But it can't skip nested `for` loop. As the dict hold the {'group1' : [user1, user2, user2], 'group2' : [user4, user5, user6] }. So this how its changed now
```python
for group, users in groups_to_users_dict.items():
        for user in users:
            membership_id = add_users_to_group(groups_for_section_class, group, user)